gangplank/cosa/build.go

```diff
@@ -35,6 +37,9 @@ var (
 
 	// ErrMetaNotFound is thrown when a meta.json cannot be found
 	ErrMetaNotFound = errors.New("meta.json was not found")
+
+	// reMetaJSON matches meta.json files use for merging
+	reMetaJSON = regexp.MustCompile(`^meta.*.json$`)
```

Comment:
Should this instead be e.g. `^meta\.[^.]+.json$`?

Reply:
Putting `` caused the test cases to fail. In the next push, I have `^meta\.(json|.*\.json)$` with another test to prove out the reg ex.